modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorBuilder.es.js

```diff
@@ -119,7 +124,8 @@ class ContributorBuilder extends React.Component {
 				return {
 					contributors
 				};
-			}
+			},
+			() => onQueryChange && onQueryChange()
```

Comment:
Would it make sense to default `onQueryChange` to a no-op? That way you can pass it by itself as the second argument.